### PR TITLE
Load hv_sock module

### PR DESCRIPTION
**Pull request by guillaumerose**
Add support for Hyper-V socket. It does nothing on Linux and macOS but load the module on Windows.

### CODE REVIEW

File: snc.sh [hunk 1]
```diff
@@ -323,6 +323,10 @@ ${OPENSHIFT_INSTALL} --dir ${INSTALL_DIR} destroy cluster ${OPENSHIFT_INSTALL_EX
 rm id_rsa_crc* || true
 ssh-keygen -N "" -f id_rsa_crc -C "core"
 
+# Adding Hyper-V socket support
```
Comment (cfergeau) [inline snc.sh:326]: s/socket/vsock ?

**Review by cfergeau — COMMENTED** [review]
Log is slightly confusing, it does not "load the module on Windows", but "load the module when the VM si being run by a HyperV hypervisor"
create_disk.sh already download some hyper-v specific rpms, maybe this should go there as well?

File: snc.sh (hunk 1 above)
Comment (guillaumerose) [reply snc.sh]: changed

**Comment by guillaumerose** [comment]
I moved the code to createdisk.sh. Although I don't see why we don't put most of the changes in snc.sh and keep createdisk.sh as minimal as possible.

**Comment by praveenkumar** [comment]
> I don't see why we don't put most of the changes in snc.sh and keep createdisk.sh as minimal as possible.

@guillaumerose the motivation is all the VM related changes are part of `createdisk.sh` and all the openshift cluster related changes are part of `snc.sh` to keep us bit sane :)

File: createdisk.sh [hunk 2]
```diff
@@ -315,6 +315,11 @@ ${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- 'sudo rpm-ostree install /home/
 # Remove the packages from VM
 ${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- rm -fr /home/core/packages
 
+# Adding Hyper-V vsock support
+${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} 'sudo bash -x -s' <<EOF
+    echo hv_sock > /etc/modules-load.d/vsock.conf
+EOF
```
Comment (praveenkumar) [inline createdisk.sh:321]: :+1:

**Review by praveenkumar — APPROVED** [review]

**Pull request merged**